Remove inappropriate "please", minor rewording (#4249)
iterative/dvc.org: content/docs/studio/troubleshooting.md

## Support

> If you need further help, feel free to send us a message using `Help` on the

Reviewer: ```suggestion
If you need further help, you can send us a message using `Help` on the
```

   To create projects for subdirectories, [specify the project directory in
   project settings].

>    If this solution does not work for your use case, don't hesitate to create a support

Reviewer: ```suggestion
   If this solution does not work for your use case, please create a support
```

You can unhide commits and branches to display them. For details, refer to
[Display preferences -> Hide commits]. However, if the missing commit/branch is

> not in the hidden commits list, then make sure to [raise a support request](#support).

Reviewer: ```suggestion
not in the hidden commits list, please [raise a support request](#support).
```

[display preferences -> hide commits]:
  /doc/studio/user-guide/projects-and-experiments/explore-ml-experiments#hide-commits

## Error: Failed to push experiment to repository

This is a non-specific error with a range of possible causes. To resolve it,

> don't forget to check:

Reviewer: ```suggestion
check that:
```

  resources.
- Whether [GitHub][gh-status], [GitLab][gl-status], or [BitBucket][bb-status]
  are experiencing service disruptions. In case of an on-premises deployment,
  do check with your administrator.

[gh-status]: https://www.githubstatus.com/
[gl-status]: https://status.gitlab.com/
[bb-status]: https://bitbucket.status.atlassian.com/

> If you get this error and none of the above applies, don't forget to

Reviewer: ```suggestion
If you get this error and none of the above applies, please
```

Check out the [Frequently Asked Questions](https://studio.iterative.ai/faq) to
see if your questions have already been answered. If you still have problems,

> don't hesitate to [contact us](#support).

Reviewer: ```suggestion
please [contact us](#support).
```
